src/components/LineChart/index.js

```diff
@@ -26,7 +27,7 @@ const LineChart = (props: Props) => {
     return moment(tickItem).format('MMM Do hh:mm');
   }
 
-  if (isLoading) return <div className="loader" />;
+  if (isLoading) return <Spinner />;
```

Comment:
Can you add a blank line between lines 30 and 31 to make it more readable? Thanks.